### PR TITLE
Allows the submission of a change-log for non-beta internal tests.

### PR DESCRIPTION
**Pull request by manlycode**
Addresses issue https://github.com/fastlane/pilot/issues/57

### CODE REVIEW

File: lib/pilot/build_manager.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def upload(options)
       if result
         Helper.log.info "Successfully uploaded the new binary to iTunes Connect"
 
-        unless config[:skip_submission]
+        unless config[:skip_submission] && config[:changelog].nil?
```
Comment (KrauseFx) [inline lib/pilot/build_manager.rb]: It's confusing to have 2 statements in one `unless`. Could you change it be "more positive"?

File: lib/pilot/build_manager.rb [hunk 2]
```diff
@@ -106,7 +106,8 @@ def distribute_build(uploaded_build, options)
       uploaded_build.update_build_information!(whats_new: options[:changelog])
 
       # Submit for internal beta testing
-      uploaded_build.build_train.update_testing_status!(true, 'external')
+      uploaded_build.build_train.update_testing_status!(!options[:skip_submission], 'external')
```
Comment (KrauseFx) [inline lib/pilot/build_manager.rb]: Wouldn't it be okay to just skip this line if `skip_submission` is false?

**Comment by KrauseFx** [comment]
Thanks for the pull request, great contribution :+1: I added 2 comments before the pull request is ready to be merged :+1:

**Comment by manlycode** [comment]
Do internal builds (29e86e22f05ed813883743467d57733d55955de9) support changelings? If so, I think this PR is moot.

**Comment by micnguyen** [comment]
Does this address issue #86 of internal builds not receiving text in "What to Test"? (Assuming change log is 'What to Test')

**Comment by squarefrog** [comment]
Has there been any traction with this PR? Is it only possible to set the `whats_new` text **after** a build has been uploaded and processed?

In iTunes Connect you're able to save text inside the What to Test block while the binary is still processing. The advantage of this is that when the new build email is sent out users are notified what to test. Otherwise they only see this information when going back into TestFlight.

**Comment by KrauseFx** [comment]
@squarefrog Yeah, as far as I know it's only possible after the build was finished processing. You can try so on iTunes Connect

**Comment by squarefrog** [comment]
@KrauseFx it **is** possible on iTunes connect while the app is processing. Not sure if this has always been the case though.

**Comment by mpirri** [comment]
@manlycode I am sorry to have to do this, but we are moving this weekend to a new, unified [fastlane mono repo](https://krausefx.com/blog/our-goal-to-unify-fastlane-tools). Since this PR is not currently in a state to be merged, I need to close it. Please feel free to open a new PR on the new repo after Monday if you are still interested in contributing this to `pilot`. Let us know if you need any assistance with this process.

**Pull request closed without merging**